### PR TITLE
Reducing duplication using block models

### PR DESCRIPTION
**Pull request by sareh**
Reduces duplication (addresses CodeClimate warnings). 

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/dataValidator/helpers/validators/validateNode.test.js [hunk 1]
```diff
@@ -209,26 +211,9 @@ describe('Validate block', () => {
...
-                    text:
-                      'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
-                  },
-                },
-              ],
-            },
-          },
-        ],
-      },
+      model: modelTextBlock(
```
Comment (pjlee11) [inline src/app/dataValidator/helpers/validators/validateNode.test.js:214]: Is there a reason we aren't using `singleTextBlock` here?

**Review by pjlee11 — COMMENTED** [review]
Just a small question, good work 👍

**Review by jtart — DISMISSED** [review]
Thanks v much for doing this!

Just one thing about `blockId`. If it is removed from fixture, the tests will always throw warnings on prop types. As they were already throwing prop type warnings, that might be something we want to consider if we get around to tidying up the warnings.

File: src/app/dataValidator/helpers/validators/validateNode.test.js (hunk 1 above)
Comment (sareh) [reply src/app/dataValidator/helpers/validators/validateNode.test.js:214]: @pjlee11 
This includes the object with the blocks array, which within it is `singleTextBlock`
```
{
  blocks: [singleTextBlock(text)]
}
```
This is the value needed for the model.

**Comment by sareh** [comment]
@jtart Thanks. Yes, we will need to remove blockIds very soon (https://github.com/bbc/simorgh/issues/661) so we'll need to make that change.

**Review by pjlee11 — APPROVED** [review]

**Review by jtart — APPROVED** [review]

**Pull request merged**